index.html

```diff
@@ -198,7 +198,7 @@ <h3>Language basics</h3>
   
   
    	<div class="req" id="lang_basics_meta">
-	<p class="advisement">Consider whether it is useful to express the <a href="https://w3c.github.io/bp-i18n-specdev/#sec_lang_meta">intended linguistic audience</a> of a resource, in addition to specifying the language used for <a href="https://w3c.github.io/bp-i18n-specdev/#sec_text_processing_lang">text processing</a>.</p>
+	<p class="advisement">Consider whether it is useful to express the <a href="#sec_lang_meta">intended linguistic audience</a> of a resource, in addition to specifying the language used for <a href="#sec_text_processing_lang">text processing</a>.</p>
```

Comment:
Consider using the ReSpec markup to create the links. When we generate the TR version, the resulting markup would end up with complete document links instead of relative links, which is useful when people copy the content.

Reply:
Well, the problem is that I'm not sure how to add link text if I use the `[[[#some-id]]]` syntax.

ReSpec will automatically generate the link text using the section name, but sometimes we do not want to use the section name as the link text.

Reply:
This suggests that _intended linguistic audience_ wants to be a term (we have _text processing language_ in the glossary: this has historically been its counterpoint). I'm not sure we want to put it into our formal glossary, but we might want to define it locally to specdev?

Reply:
OK. I just defined it in specdev.